src/generate/register.rs

```diff
@@ -414,7 +484,9 @@ pub fn fields(
                 if Some(evs) != evs_r.as_ref() {
                     pc_w = &new_pc_w;
                     if let Some(base) = base {
-                        let pc = base.field.to_sanitized_upper_case();
+                        let pc = util::replace_suffix(base.field, "")
+                            .to_sanitized_upper_case()
+                            .to_string();
```

Comment:
Do you need this `.to_string();`?

Reply:
> ```rust
> CH_ABORT_R
> ```

This field does not contain <dim> 
https://github.com/riscv-rust/k210-pac/blob/4f7330528951b6d1f54ed5030d7f17b0078cdce5/k210.svd#L1123
Please add there:
```
                            <dim>6</dim>
                            <dimIncrement>1</dimIncrement>
                            <dimIndex>1-6</dimIndex>
```